docs/schemas-tests/.editorconfig

```diff
@@ -0,0 +1,34 @@
+# see https://aka.ms/editorconfigdocs
+
+# We make this the root, so we can test what the actual .editorconfig file would be for
+# a customer's repo:
+root = true
+
+# Explicit config for yaml files
+[*.{yaml,yml}]
```

Comment:
Yaml files are in other directories as well. Let's move this to root config

Reply:
this file is here explicitly for authoring of schemas.
I've moved it in my latest iteration, but it's intended to allow opening the schemas-tests folder in a separate VSCode window so that changes to schema are faster to load.